### PR TITLE
Replaced all "melee", "bullet" with MELEE, BULLET etc [MDB ignore]

### PR DESCRIPTION
**Pull request by Zar000**
## About The Pull Request

Tivi has created defines and they should be used

## Why It's Good For The Game

Define good
## Changelog
:cl:
fix: define better than string
/:cl:

### CODE REVIEW

File: _maps/map_files/Sulaco/Sulaco.dmm [hunk 1]
```diff
@@ -23693,7 +23693,7 @@
 	desc = "A head-mounted face cover designed to protect the wearer completely from space-arc eye. This one appears reinforced to better protect the cranium. Rumored to have been made to thwart the efforts of sneaky Runers and hunters. A small inscription reads: 'D.C'";
 	flags_armor_features = 16;
 	name = "Modified welding helmet";
-	soft_armor = list("melee"=15,"bullet"=0,"laser"=0,"energy"=0,"bomb"=0,"bio"=0,"rad"=0,"fire"=0,"acid"=0)
+	soft_armor = list(MELEE = 15, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, "rad" = 0, FIRE = 0, ACID = 0)
```
Comment (TiviPlus) [inline _maps/map_files/Sulaco/Sulaco.dmm:23696]: What the fuck

File: _maps/modularmaps/EORG/de_inferno.dmm [hunk 2]
```diff
@@ -1206,7 +1206,7 @@
 	name = "Blocker"
 	},
 /turf/closed/wall/wood{
-	soft_armor = list("melee" = 1000, "bullet" = 1000, "laser" = 1000, "energy" = 1000, "bomb" = 1000, "bio" = 1000, "rad" = 1000, "fire" = 1000, "acid" = 1000)
+	soft_armor = list(MELEE = 1000, BULLET = 1000, LASER = 1000, ENERGY = 1000, BOMB = 1000, BIO = 1000, "rad" = 1000, FIRE = 1000, ACID = 1000)
```
Comment (TiviPlus) [inline _maps/modularmaps/EORG/de_inferno.dmm:1209]: who

File: code/game/objects/items/reagent_containers/food/snacks.dm [hunk 3]
```diff
@@ -641,7 +641,7 @@
 	icon_state = "xenomeat"
 	filling_color = "#43DE18"
 	list_reagents = list(/datum/reagent/consumable/nutriment = 3)
-	tastes = list("meat" = 1, "acid" = 1)
+	tastes = list("meat" = 1, ACID = 1)
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/food/snacks.dm:644]: check your autoreplace

File: code/game/objects/items/reagent_containers/food/snacks.dm [hunk 4]
```diff
@@ -771,7 +771,7 @@
 	filling_color = "#43DE18"
 	list_reagents = list(/datum/reagent/consumable/nutriment = 8)
 	bitesize = 2
-	tastes = list("bun" = 4, "acid" = 4)
+	tastes = list("bun" = 4, ACID = 4)
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/food/snacks.dm:774]: check your autoreplace

File: code/game/objects/items/reagent_containers/food/snacks.dm [hunk 5]
```diff
@@ -930,7 +930,7 @@
 	trash = /obj/item/trash/plate
 	filling_color = "#43DE18"
 	list_reagents = list(/datum/reagent/consumable/nutriment = 10)
-	tastes = list("pie" = 1, "meat" = 1, "acid" = 1)
+	tastes = list("pie" = 1, "meat" = 1, ACID = 1)
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/food/snacks.dm:933]: check your autoreplace

File: code/game/objects/items/reagent_containers/food/snacks.dm [hunk 6]
```diff
@@ -1811,7 +1811,7 @@
 	slice_path = /obj/item/reagent_containers/food/snacks/xenomeatbreadslice
 	list_reagents = list(/datum/reagent/consumable/nutriment = 30)
 	filling_color = "#8AFF75"
-	tastes = list("bread" = 10, "acid" = 10)
+	tastes = list("bread" = 10, ACID = 10)
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/food/snacks.dm:1814]: check your autoreplace

File: code/game/objects/items/reagent_containers/food/snacks.dm [hunk 7]
```diff
@@ -1821,7 +1821,7 @@
 	trash = /obj/item/trash/plate
 	filling_color = "#8AFF75"
 	bitesize = 2
-	tastes = list("bread" = 10, "acid" = 10)
+	tastes = list("bread" = 10, ACID = 10)
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/food/snacks.dm:1824]: check your autoreplace

File: code/modules/client/preferences_toggles.dm [hunk 8]
```diff
@@ -212,7 +212,7 @@ GLOBAL_LIST_INIT(ghost_forms, list("Default" = GHOST_DEFAULT_FORM, "Ghost Ian 1"
 							"Cyan" = "ghost_cyan", "Dark Blue" = "ghost_dblue", "Dark Red" = "ghost_dred", "Dark Green" = "ghost_dgreen", \
 							"Dark Cyan" = "ghost_dcyan", "Grey" = "ghost_grey", "Dark Yellow" = "ghost_dyellow", "Dark Pink" = "ghost_dpink",\
 							"Purple" = "ghost_purpleswirl", "Funky" = "ghost_funkypurp", "Transparent Pink" = "ghost_pinksherbert", "Blaze it" = "ghost_blazeit",\
-							"Mellow" = "ghost_mellow", "Rainbow" = "ghost_rainbow", "Camo" = "ghost_camo", "Fire" = "ghost_fire", "Cat" = "catghost"))
+							"Mellow" = "ghost_mellow", "Rainbow" = "ghost_rainbow", "Camo" = "ghost_camo", FIRE = "ghost_fire", "Cat" = "catghost"))
```
Comment (TiviPlus) [inline code/modules/client/preferences_toggles.dm:215]: check your autoreplace

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Priority merge due to conflict potential

File: _maps/map_files/Sulaco/Sulaco.dmm (hunk 1 above)
Comment (Zar000) [reply _maps/map_files/Sulaco/Sulaco.dmm:23696]: Do I just delete this shit or wtf I do

File: _maps/modularmaps/EORG/de_inferno.dmm (hunk 2 above)
Comment (Zar000) [reply _maps/modularmaps/EORG/de_inferno.dmm:1209]: bild

File: _maps/map_files/Sulaco/Sulaco.dmm (hunk 1 above)
Comment (TiviPlus) [reply _maps/map_files/Sulaco/Sulaco.dmm:23696]: Yes

File: _maps/modularmaps/EORG/de_inferno.dmm (hunk 2 above)
Comment (TiviPlus) [reply _maps/modularmaps/EORG/de_inferno.dmm:1209]: this one too

**Pull request merged**